### PR TITLE
fix(Turntable): black screen and the song preview disappear when toggle the full app display

### LINKED ISSUE #714: [Turntable] Unable to exit the turntable. ESC not working.
**Describe the bug**
When I tried to exit the turntable using the ESC button, the entire app will freeze into a black screen. I have to restart the entire app to make it work.

**To Reproduce**
1. Config turntable extension
2. Play a song
3. Click on the Turntable button on the top left of the screen
4. Try to exit the Turntable with ESC

**Expected behavior**
After clicking ESC, the turntable should be exit and return to the normal page.

**Specifics (please complete the following information):**
* Spotify version [1.1.81.604.gccacfc8c]
* Spicetify version [2.9.4]
* Theme name: Turntable

### PR DESCRIPTION
**Pull request by grasonchan**
fix #714

### CODE REVIEW

File: Turntable/turntable.js [hunk 1]
```diff
@@ -303,23 +290,23 @@ window.addEventListener("load", rotateTurntable = () => {
...
       fullAppDisplay.addEventListener("contextmenu", () => handleContextMenu(fullAppDisplay), { once: true });
 
-      fullAppDisplay.addEventListener("dblclick", () => handleMainInterface());
+      // fullAppDisplay.addEventListener("dblclick", () => handleToggleFad());
 
       clickedFadBtn = true;
     }
 
-    handleMainInterface("active", topbarContentFadeIn);
+    // handleToggleFad(true);
```
Comment (afonsojramos) [inline Turntable/turntable.js:309]: Are both of these commented on purpose? The function never gets called this way 🤔

**Comment by grasonchan** [comment]
> Yeah, you are clearly trying to fight the bot. 😅 I think I'll remove the labeling and just use the assigning.

Yep! The bot automatically add the `help wanted` lable every time when i do an action, but i don't need it. 😂

Does the bot add lable based on the title? And sorry for the late reply.

File: Turntable/turntable.js (hunk 1 above)
Comment (grasonchan) [reply Turntable/turntable.js:309]: That's legacy code. I want to quickly process invalid code fisrt, and i will change it later.

**Comment by afonsojramos** [comment]
> Does the bot add lable based on the title?

Not anymore :D

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**